hypervisor/dm/vpci/pci_pt.c

```diff
@@ -195,6 +195,33 @@ void vdev_pt_write_vbar(struct pci_vdev *vdev, uint32_t idx, uint32_t val)
 	}
 }
 
+void vdev_pt_do_flr(const struct pci_vdev *vdev, uint32_t offset, uint32_t bytes, uint32_t val)
```

Comment:
This API may be used by hypervisor device too. How about to name it as pdev_do_flr ?

Reply:
OK.